.github/workflows/release.yml

```
@@ -55,10 +55,14 @@ jobs:
         uses: actions/checkout@v3
         with:
           path: ${{ env.IMGTOOL_PACKING_PATH }}
+        
+      - name: Set the version
+        working-directory: ${{ env.IMGTOOL_PACKING_PATH }}/patches/
+        run: perl -pi.bak -e "s/ARDUINO_VERSION_PLACEHOLDER/${GITHUB_REF/refs\/tags\//}/g" 0008-Imgtool-Append-arduino-to-version-string.patch
```

```suggestion
        run: perl -pi -e "s/ARDUINO_VERSION_PLACEHOLDER/${GITHUB_REF/refs\/tags\//}/g" 0008-Imgtool-Append-arduino-to-version-string.patch
```
There is no need to make a backup file that won't be used.